src/PowerShellEditorServices/Utility/PathUtils.cs

```diff
@@ -29,13 +30,48 @@ internal static class PathUtils
         /// </summary>
         internal static readonly char AlternatePathSeparator = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? '/' : '\\';
 
+        /// <summary>
+        /// The <see cref="StringComparison" /> value to be used when comparing paths. Will be
+        /// <see cref="StringComparison.Ordinal" /> for case sensitive file systems and <see cref="StringComparison.OrdinalIgnoreCase" />
+        /// in case insensitive file systems.
+        /// </summary>
+        internal static readonly StringComparison PathComparison = RuntimeInformation.IsOSPlatform(OSPlatform.Linux)
```

Comment:
Platform checks are sort of littered throughout the project. It'd be nice to go through and replace these with centralized logic. (It's optimized by the JIT anyway iirc so it's not a perf boost at least outside of Windows PowerShell, but nice for maintenance)

Reply:
💯 agree!